.github/workflows/benchmark.yml

```diff
@@ -17,6 +17,7 @@ on:
     branches:
       - stable
       - unstable
+  workflow_dispatch:
```

Comment:
is this necessary?

Reply:
I think it is just to be able to trigger it manually, might be useful to have

Reply:
In some discussions @dapplion suggested to open draft PRs to test some idea. For that purpose it's required sometime to run some workflows manually on branches.